test/90_1_prm/data/space_envelope_areas.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
not sure why this file is necessary

Reply:
It is in test/90_1_prm/data. Is that not considered the testing directory?